src/main/java/com/strandls/cca/service/impl/CCATemplateServiceImpl.java

```diff
@@ -262,6 +263,30 @@ public List<CCAField> getFilterableFields(HttpServletRequest request, String sho
 		return ccaFields;
 	}
 
+	@Override
+	public List<CCAField> getFilterableChartFields(HttpServletRequest request, String shortName, String language) {
+		if (shortName == null || "".equals(shortName))
+			shortName = CCAConstants.MASTER;
+
+		CCATemplate ccaTemplate = getCCAByShortName(shortName, language, false);
+		Iterator<CCAField> it = ccaTemplate.iterator();
+		List<CCAField> ccaFields = new ArrayList<>();
+
+		// hardcoded to get the needed aggregate
+		List<String> filteredIds = Arrays.asList("f4d0c3af-02ba-4891-b1cb-bd1a81bd0ff4",
```

Comment:
please avoid hard coding it like this. Refactor it somewhere in a constants file or something.

Reply:
for first cut , it is enough , it will be soon removed when we add isChartable filed in cca template.